conditionals.js

```diff
@@ -87,14 +87,43 @@ console.log(lightStatus(255) === "on");
 
 function getLightBulbStatusDisplayString(status) {
     let result = "";
-    /* uncomment and complete
-    switch( your code here ) {
-      case "your code here": 
-        your code here;
-        break;
-      etc...
+    switch(status){
+        case "on":
+            return "The house is bright!";
+            break;
+
+        case "off":
+            return "The house is dark";
+            break;
+
+        case "dimmed":
+            return "The house is nice and dim";
+            break;
+
+        // case "offline | | missing":
+        //     return "The house is dark and we can't find the lightbulb!";
+        //     break;
+        //why won't this work?
+
+        case "offline":
+            return "The house is dark and we can't find the lightbulb!";
+            break;
+
+        case "deleted":
+            return "The lightbulb has been removed from the system";
+            break;
+
+        case "missing":
```

Comment:
instead of repeating the same code in this block as the "offline" block, you could just put the `case` statement for each item one after another like this:
`case "offline:"`
`case: "missing":`
`    return "The house is dark and we can't find the lightbulb!";`
`    break;`